dcos/marathon.py

```diff
@@ -87,8 +88,9 @@ class Client(object):
     :type marathon_url: str
     """
 
-    def __init__(self, marathon_url):
```

Comment:
Why is this a part of the marathon interface?  We're always just using the default, so there's no need to customize it.